src/solver/libModelObject/include/antares/solver/libObjectModel/valueType.h

```diff
@@ -31,4 +33,25 @@ enum class ValueType
     BOOL
 };
 
+inline std::ostream& operator<<(std::ostream& os, const ValueType& value_type)
+{
+    using namespace std::string_literals;
+    switch (value_type)
+    {
+    case ValueType::FLOAT:
+        os << "FLOAT"s;
+        break;
+    case ValueType::INTEGER:
+        os << "INTEGER"s;
+        break;
+    case ValueType::BOOL:
+        os << "BOOL"s;
+        break;
+    default:
+        os << "UNKNOWN"s;
+        break;
+    }
+    return os;
+}
+
```

Comment:
Why is this function needed ?

Reply:
For google test message when comparing values